src/Cake.Testing/Cake.Testing.csproj

```diff
@@ -24,6 +24,7 @@
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
     <CodeAnalysisRuleSet>..\Cake.ruleset</CodeAnalysisRuleSet>
+    <LangVersion>default</LangVersion>
```

Comment:
As there's a PR for  ` <LangVersion>5</LangVersion>` this change feels unnecessary, right?